internal/characters/zhongli/stele.go

```diff
@@ -21,7 +21,7 @@ func (c *char) newStele(dur int, max int) {
 		Mult:       skill[c.TalentLvlSkill()],
 		FlatDmg:    0.019 * c.MaxHP(),
 	}
-	c.Core.QueueAttack(ai, combat.NewCircleHit(c.Core.Combat.Player(), 2), 0, 0)
+	c.Core.QueueAttack(ai, combat.NewCircleHitOnTarget(c.Core.Combat.Player(), nil, 2), 0, 0)
 
```

Comment:
Should we create some measured offset away from player rather than spawn on player? Have position passed into `newStele` to ensure same position as hold damage instance? maybe y offset is supposed to be `4` here?

Reply:
Y offset is supposed to be `3` to line up with stele position. I'll change it.